decayAmplitude/isobarAmplitude.h

```diff
@@ -74,6 +74,8 @@ namespace rpwa {
 		void enableSpaceInversion(const bool flag = true) { _doSpaceInversion = flag; }  ///< en/disables parity transformation of decay
 		void enableReflection    (const bool flag = true) { _doReflection     = flag; }  ///< en/disables reflection of decay through production plane
 
+		void setFromXDecay() {setDecayTopology(rpwa::createIsobarDecayTopology(decayTopology()->subDecayConsistent(decayTopology()->XIsobarDecayVertex())));} ///< Sets a decay topology that only needs final-state momenta and assumes to be in the GJ frame
```

Comment:
`setTopologyFromXDecay` or `setTopologyToXDecay` might be a more informative name?